must-gather/collection-scripts/gather

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+mkdir -p /must-gather/
+
+# generate /must-gather/version file
+DIR_NAME=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+. ${DIR_NAME}/version
+echo "node-maintenance-operator/must-gather" > /must-gather/version
+version >> /must-gather/version
+
+# get namespace of node-maintenance-operator
+NMO_NAMESPACE=$( oc get pods -A -l name='node-maintenance-operator' -o=jsonpath='{.items[0].metadata.namespace}{"\n"}' )
+# fallback to default
+[ -z "${NMO_NAMESPACE}" ] && NMO_NAMESPACE="openshift-operators"
+
+# TODO get nmo logs
```

Comment:
Why `TODO`? Does something is missing?

Reply:
we might want to gather NMO logs, but I didn't have time to check how to do that. We can add that as a follow up. It's more important to have something ready for the ds builds quickly.